doc/howto/usage/k8s/python/README.md

```diff
@@ -0,0 +1,35 @@
+# PaddlePaddle Cluster
+
+为了使用户不必学习Kubernetes中复杂的YAML配置以及各种概念，降低PaddlePaddle分布式训练的复杂程度，基于[Kubernetes Python Client](https://github.com/kubernetes-incubator/client-python) 提供Python版本的分布式训练API.
+
+- 环境准备
+通知系统管理在Kubernetes集群中根据不同的分布式存储类型创建PersistentVolume以及PersistentVolumeClaim，并得到创建的PersistentVolumeClaim,得到一个分布式存储的卷。
+
+
+- 初始化集群配置
+
+  ```python
+  import paddle
+  from paddle import PaddlePaddleCluster
+  import time
+  paddle_cluster = PaddlePaddleCluster(namespace="paddle", name="paddle-cluster-job")
```

Comment:
It doesn't look like a Python class constructor can construct a "cluster"; it is we who build the cluster. It seems a "job" instead of a "cluster" here.  If it is, `PaddlePaddleCluster` ==> `Job`.